winspool/cairo.go

```diff
@@ -1,3 +1,5 @@
+// +build linux darwin
```

Comment:
1) This should not be necessary. Please remove this build tag and try to build gcp-cups-connector and gcp-connector-util again (do not try to build gcp-windows-connector)
2) If it were necessary, then it would be `+build windows`
3) Same goes for everything in winspool